lib/dyph3/two_way_differs/output_converter.rb

```diff
@@ -9,6 +9,27 @@ def convert_to_dyph3_output(old_text, new_text)
         correct_offsets(selected_actions)
       end
 
+      def objectify(merge_results)
+        merge_results.map do |result|
+          action = result[:action]
+          line   = result[:line]
+          old_index = result[:old_index]
+          new_index = result[:new_index]
+
+          case action
+          when :add
+            #:action=>:add, :line=>"e", :old_index=>4, :new_index=>5}]
+            Dyph3::Add.new(value: line, old_index: old_index, new_index: new_index)
+          when :delete
+            Dyph3::Delete.new(value: line, old_index: old_index, new_index: new_index)
+          when :no_change
+            Dyph3::NoChange.new(value: line.text, old_index: old_index, new_index: new_index)
```

Comment:
@aamontal Remind me again why it's valuable to have the NoChange wrap the original value, instead of just having a naked value. What does the old_index and new_index on non-changes allow us to build?